Resources/Prototypes/_White/Entities/Objects/Weapons/Guns/Ship/weapons.yml

```
@@ -0,0 +1,315 @@
+  id: WeaponShipLightIR
+  name: IR Pulse Laser
+  description: A power-efficient, albeit weak infrared pulse laser. Being essentially a scaled-up version of a regular laser carbine, this cannon trades damage for extreme power efficiency.
+  components:
+  - type: Sprite
+    sprite: _White/Objects/Weapons/Guns/Turrets/IR_laser.rsi
+    state: icon
+  - type: Gun
+    fireRate: 3.5
+    selectedMode: FullAuto
+    availableModes:
+      - FullAuto
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)